pennylane/_device.py

```diff
@@ -631,14 +631,20 @@ def batch_transform(self, circuit):
         """
 
         # If the observable contains a Hamiltonian and the device does not
-        # support Hamiltonians, or if the simulation uses finite shots,
+        # support Hamiltonians, or if the simulation uses finite shots, or
```

Comment:
If I recall correctly, `expval(H)` is supported only in `default.qubit`. Wondering if the current addition is general for all devices?